fe/fe-core/src/main/java/com/starrocks/service/FrontendOptions.java

```diff
@@ -101,6 +101,9 @@ public static InetAddress getLocalHost() {
     }
 
     public static String getLocalHostAddress() {
+        if (Config.enable_fqdn) {
```

Comment:
We should deprecate the logic of picking localAddr if Config.enable_fqdn is true.

Reply:
When there are multiple NICs in a machine, the priority_networks parameter still needs to be used, and in a machine with multiple NICs, one IP needs to be selected as the localAddr of the current machine, and the FQDN of the corresponding IP needs to be configured when configuring all /etc/hosts in the cluster

Reply:
You can use InetAddress.getLocalHost().getHostName() to get fqdn